Helper/ShipmentOptions.php

```diff
@@ -220,8 +229,8 @@ public static function getValueFromAttribute(
      */
     public static function getValueOfOptionWhenSet(string $key, array $options): ?bool
     {
-        if ($options[$key] || array_key_exists($key, $options)) {
-            return (bool)$options[$key];
+        if ($options[$key] || false === $options[$key]) {
```

Comment:
Dit gaat op allerlei manieren fout als `$key` niet in `$options` zit, wat blijkbaar mag gebeuren omdat deze functie public is. Plus, wat is hier de bedoeling van? 

Åls hij truthy of niet false is returnt hij null? Dus alleen lege string en lege array returnen null? En `$options[$key]` hoeft helemaal niet per se een boolean te zijn maar de functie verwacht wel altijd een boolean of null.

Kijk hier nog even goed naar, dit gaat websites omver trekken 😛